### DISCUSSION TITLE
Microsoft Search - AutoPreviewImageUrl is null

### DISCUSSION THREAD

**Discussion by Lotsamocha01** [microsoft-search/pnp-modern-search#2995] [Q&A]
When using a Search Result panel with the SharePoint Search data source the AutoPreviewImageUrl is working correctly. If I change the data source to Microsoft Search, the AutoPreviewImageUrl is null for all documents. The "Show Thumbnail" switch is On and I'm using a List template.

**Comment by kasperbolarsen** [comment]
Can you verify if it is an API issue by running the same query in the Graph Explorer?

**Reply by Lotsamocha01** [reply]
Could I get a little more info on running this query? Is Graph Explorer an online tool or another project on Github?

**Reply by kasperbolarsen** [reply]
As I remember it this video is a good primer :https://youtu.be/KAsyBgKf-RU
